talibs/ta/ta_main.c

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#include <stdio.h>
+#include <talib1.h>
+#include <talib2.h>
+#include <talib3.h>
+#include <tee_internal_api.h>
+
+TEE_Result TA_CreateEntryPoint(void)
+{
+	printf("TA_CreateEntryPoint()\n");
```

Comment:
`DMSG()`?

Reply:
For sure `xMSG()` are convenient and are already used in other examples, but IMO `printf()` is simpler and does not depend on setting the proper CFG_TEE_TA_LOG_LEVEL flag.

Reply:
printf() is not referenced in the op-tee dev doc. So maybe removed one day.
So I think it's a bad idea in the **optee_examples** repo, even if quite convenient when hacking a bit.

Reply:
note: ok for `printf`